README.md

```diff
@@ -50,17 +50,28 @@ If the script is run by Abracadabra, it looks for the magic line "Image built
 in (path)", so don't change that (without changing the other).
 
 ## Sage Users
-Follow these steps in order to create a Chameleon instance that can be configured with a Chameleon-Sage-Image and with a Docker group that allows a user cc to run Docker command without ```sudo``` priveledges.
+Follow these steps in order to create a Chameleon instance that can be configured with a Chameleon-Sage-Image and with a Docker group that allows a user cc to run Docker commands without ```sudo``` .
```

Comment:
I think it would be better to clarify that this is the process for creating an image to use for a Chameleon instance, and not a process for creating a Chameleon instance.